pages/factions/[type].tsx

```diff
@@ -32,7 +32,13 @@ const Faction: NextPage<{ faction: FactionDTO; terms: TermsDTO }> = ({
         </Stack>
 
         <Title order={2}>{terms.wielders}</Title>
-        <Stack>
+        <SimpleGrid
+          breakpoints={[
+            { minWidth: 'sm', cols: 1 },
+            { minWidth: 'md', cols: 2 },
+            { minWidth: 1200, cols: 3 },
```

Comment:
Couldn't we use defined breakpoints like `lg` or create a new breakpoint?
See https://mantine.dev/theming/responsive/